akka-typed-testkit/src/main/scala/akka/typed/testkit/Effects.scala

```diff
@@ -20,7 +20,7 @@ import scala.concurrent.duration.{ Duration, FiniteDuration }
 abstract class Effect
 
 object Effect {
-  @SerialVersionUID(1L) final case class Spawned(childName: String) extends Effect
+  @SerialVersionUID(1L) final case class Spawned(childName: String, props: Option[Props]) extends Effect
```

Comment:
Hmmm.  Shouldn't that be a different effect? (Sorry if I'm potentially making this nice little PR grow ;) )